src/fluxo.object_store.js

```diff
@@ -194,20 +123,21 @@ class ObjectStore {
   listenStoreAttribute (attribute, store) {
     let onStoreEvent = function(attributeName, eventName, ...args) {
       if (eventName !== "change") {
-        this.triggerEvent(
+        this.radio.triggerEvent(
           `change:${attributeName}:${eventName.replace(/^change:/, '')}`,
+          this,
```

Comment:
You will see the `this` over all the file because we have a contract that the callback will receive the instance as argument.

We must decide if this intended to be kept or the callback being binded to the store will be enough.

I prefer to not bind the callback and always have the store as the second argument (that would be a breaking change but we must define a contract because the events already differ from each other in that sense, I can open a second pull request to define that and latter we can go back here and have this decided).